provisioning/AWS/aws-resources.yaml

```diff
@@ -0,0 +1,114 @@
+Parameters:
+  GitHubOrg:
+    Type: String
+  RepositoryName:
+    Type: String
+  OIDCProviderArn:
+    Description: Arn for the GitHub OIDC Provider.
+    Default: ""
+    Type: String
+  OIDCAudience:
+    Description: Audience supplied to configure-aws-credentials.
+    Default: "sts.amazonaws.com"
+    Type: String
+
+Conditions:
+  CreateOIDCProvider: !Equals
+    - !Ref OIDCProviderArn
+    - ""
+
+Resources:
+  CommonDriverStorageBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: ci-storage-driver-common
+  StorageDriverCommonRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Action: sts:AssumeRoleWithWebIdentity
+            Principal:
+              Federated: !If
+                - CreateOIDCProvider
+                - !Ref GithubOidc
+                - !Ref OIDCProviderArn
+            Condition:
+              StringEquals:
+                token.actions.githubusercontent.com:aud: !Ref OIDCAudience
+              StringLike:
+                token.actions.githubusercontent.com:sub: !Sub repo:${GitHubOrg}/${RepositoryName}:*
+      Policies:
+        - PolicyName: DownloadDrivers
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Action:
+                  - s3:ListBucket
+                  - s3:GetObject
+                Resource:
+                  - arn:aws:s3:::keboola-drivers/*
+                  - arn:aws:s3:::keboola-drivers
+                Effect: Allow
+        - PolicyName: FileStorage
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Action:
+                  - s3:ListBucket
+                  - s3:GetObject
+                  - s3:PutObject
+                Resource:
+                  - Fn::GetAtt: [ CommonDriverStorageBucket, "Arn" ]
+                  - !Join
```

Comment:
tady to jde udelat i jednoduseji pomoci `Sub` https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-sub.html#w2ab1c33c28c64c11 `Fn::Sub without a mapping`

`!Sub 'arn:aws:s3:::${CommonDriverStorageBucket}/*'`

Reply:
Jo diky take neco som hladal :) upravil som to 👍